pyproject.toml

```diff
@@ -72,7 +72,7 @@ test = [
     "pytest-mock>=1.7.1, <4.0",
     "pytest-xdist[psutil]~=2.2.1",
     "pytest>=7.2,<9.0",
-    "s3fs>=2021.4, <2024.6",  # Upper bound set arbitrarily, to be reassessed in early 2024
+    "s3fs>=2021.4, <2024.7",  # Upper bound set arbitrarily, to be reassessed in early 2024
```

Comment:
```suggestion
    "s3fs>=2021.4, <2025.1",  # Upper bound set arbitrarily, to be reassessed in late 2024
```

Reply:
The comment was written last year and we bumped it a few months in advance, should we do this again?

Reply:
Ah yes makes sense!